### ISSUE TITLE
Test contribution guide

### ISSUE THREAD

**Issue by amitkrout** [code-ready/crc#1789]
### General information

  * OS: Linux / macOS / Windows
  * Hypervisor: KVM / Hyper-V / hyperkit
  * Did you run `crc setup` before starting it (Yes/No)? - NA

## CRC version
```bash
# Put the output of `crc version`
```
I am on master branch  
## CRC status
```bash
# Put the output of `crc status`
```
NA 
## CRC config
```bash
# Put the output of `crc config view`
```
VA
## Host Operating System
```bash
# Put the output of `cat /etc/os-release` in case of Linux
# put the output of `sw_vers` in case of Mac
# Put the output of `systeminfo` in case of Windows
```
NA
### Steps to reproduce

No test contribution guide is available on master branch  

### Expected

There should be a test contribution guide on master branch that should touch like how to setup the test env (both unit and integration) and run it.

### Actual
No doc i can see for integration test on master

### Logs

You can start crc with `crc start --log-level debug` to collect logs.
Please consider posting this on http://gist.github.com/ and post the link in the issue.

**Comment by praveenkumar** [comment]
@amitkrout https://github.com/code-ready/crc/blob/master/developing.adoc doesn't this help, if not can you please let us know which step didn't work or their is any missing info in this doc.

**Comment by amitkrout** [comment]
@praveenkumar My bad i was searching it in ```docs/``` dir. Thanks for sharing the the link. I was looking thsi doc for setting up the crc test env and run a basic test scenario just to understand how it works with https://github.com/code-ready/clicumber.

I will open up new issue in case i found any difficulty to setup the test env or for any missing steps.
